app/controllers/tasks_controller.rb

```diff
@@ -29,7 +29,7 @@ def edit
 
   def update
     @task = Task.find(params[:id])
-    if @task.update_attributes task_params
+    if @task.update task_params
```

Comment:
the fn has been renamed in newer rails version https://rubyinrails.com/2019/04/09/rails-6-1-activerecord-deprecates-update-attributes-methods/

Reply:
Replaced at all places.